stacks/kafka-flink/README.rst

```diff
@@ -36,26 +36,25 @@ About
 This tutorial shows how to build a simple data ingestion pipeline with open
 source software components.
 It will outline how to acquire and publish data to Kafka, process it using
-Flink and store the data stream into CrateDB.
+Flink, and store the data stream into CrateDB.
 It describes this system as a part of the CrateDB reference architecture and
 can be used as a blueprint for building own applications.
 
 The following versions of software components are used:
 
-- Confluent Kafka 6.1.1
 - Apache Flink 1.12
```

Comment:
Just a reminder to sync with: https://github.com/crate/cratedb-examples/pull/23 regarding the updated versions.

Reply:
Yeah. Bumped to 6.1.9 there - thanks!